netcdf_to_gltf_converter/netcdf/netcdf_data.py

```diff
@@ -189,15 +248,18 @@ def fill_value(self) -> int:
         pass
     
     @abstractmethod
-    def shift_coordinates(self, shift_x: float, shift_y: float) -> None:
+    def shift_coordinates(self, shift_x: float, shift_y: float, shift_z: float, variables: List[str]) -> None:
         """
-        Shift the x- and y-coordinates in the data set with the provided values.
+        Shift the x- and y-coordinates and the variable values in the data set with the provided values.
```

Comment:
what is the shift_z exactly, I was in the assumption it was another dimension, but in the documentation you state variable values.

Reply:
z = data